gitstatus/gitstatus.plugin.zsh

```diff
@@ -274,8 +274,10 @@ function gitstatus_start() {
       exec {stderr_fd}>&2 2>$xtrace_file
       setopt xtrace
     }
-
-    local os && { [[ $(uname -o) == Android ]] && os=Android || os=$(uname -s) } && [[ -n $os ]]
+    
+    # uname -o fails on macOS, so suppress it's stderr output as that case
```

Comment:
Sorry about breaking your setup and thanks for the PR.

Please do it like this instead:

```zsh
local os && os=$(uname -s) && [[ -n $os ]]
[[ $os != Linux || $(uname -o) != Android ]] || os=Android
```

Reply:
My apologies for handling it less gracefully than desired I couldn't wait any longer for your reply (the fix is kind of an emergency), so I submitted https://github.com/romkatv/gitstatus/commit/708e9a6a77af608cd10aa3069046109a9e4dc40f upstream and pulled it to powerlevel10k. There shouldn't be no warning now.

Reply:
Hi @romkatv that is perfectly fine. I was AFK for lunch and did not not expect such a quick reaction.